manifests/w/WarcraftLogs/Uploader/5.9.5.162/WarcraftLogs.Uploader.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: WarcraftLogs.Uploader
+PackageVersion: 5.9.5.162
+PackageLocale: en-US
+Publisher: RPGLogs
+PublisherUrl: https://www.warcraftlogs.com/
+PublisherSupportUrl: https://articles.warcraftlogs.com/help/getting-started
+PrivacyUrl: https://www.warcraftlogs.com/help/privacy
+Author: RPGLogs
+PackageName: Warcraft Logs Uploader
+PackageUrl: https://github.com/RPGLogs/Uploaders-warcraftlogs

```

Comment:
```suggestion
PackageUrl: https://github.com/RPGLogs/Uploaders-warcraftlogs/releases
```

Reply:
@stephengillie, would https://github.com/RPGLogs/Uploaders-warcraftlogs not be correct?

> [This key](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.2.0/defaultLocale.md) represents the web site for the package.

Reply:
@russelbanks the current PackageUrl value has sparse content. The suggested PackageUrl contains more information about the package.